pycytominer/tests/test_cyto_utils/test_load.py

```diff
@@ -196,3 +197,24 @@ def test_load_npz():
         load_npz_locations(
             example_npz_file_locations, location_x_col_index=0, location_y_col_index=2
         )
+
+def test_infer_plate_files():
+
+    # inferring parquet files
+    parquet_file = "../test_data/cytominer_database_example_data/test_SQ00014613.parquet"
+    expected_file_type_1 = "parquet"
+    test_file_type_infer_1 = infer_profile_file_type(parquet_file)
+    assert(expected_file_type_1 == test_file_type_infer_1)
```

Comment:
This is mostly up to your preference and I wanted to make a suggestion here for simplicity. Consider making direct references instead of declaring variables for items which are used only once (this stands for some other lines below as well). For example, see this suggestion:
```suggestion
    assert(infer_profile_file_type(parquet_file) == "parquet")
```